app/api/electrumAddressApi.js

```diff
@@ -353,6 +353,57 @@ function lookupTxBlockHash(txid) {
 	});
 }
 
+// Lookup the spending transaction and height of a given transaction output. Only works with Electrum 1.5 protocol, ElectRS 0.9.0 does not implement subscriptions
+// but can return the transaction spending the given outpoint
+function lookupOutpointsTx(outpoints) {
+	if (electrumClients.length == 0) {
+		return Promise.reject({ error: "Not supported by Electrum 1.4", userText: noConnectionsErrorText });
+	}
+
+	if (outpoints.length == 0) {
+		return Promise.resolve(null);
+	}
+	return runOnAllServers(function(electrumClient) {
+		return new Promise((resolve, reject) => {
+			let contents = [];
+			let arguments_far_calls = {};
+			for (let outpoint of outpoints) {
+				const id = ++electrumClient.id;
+				const request = utils.makeRequest('blockchain.outpoint.subscribe', outpoint, id);
+				contents.push(request);
+				arguments_far_calls[id] = outpoint;
+			}
+			const content = '[' + contents.join(',') + ']';
```

Comment:
Why do you use string manipulation instead of native Json encoding?

Reply:
The honest answer is that this is how it was made fore `requestBatch` in `electrum-client` lib x')

The `write` method for net.socket must take a string https://nodejs.org/api/net.html#net_socket_write_data_encoding_callback